shell/initialize/client.js

```diff
@@ -623,6 +653,7 @@ function fixPrepatch(to, ___) {
 
     if (triggerScroll) {
       // Ensure to trigger scroll event after calling scrollBehavior
+      // what's is this scroll thing?
```

Comment:
This is ultimately to force the page to scroll to  the top when transitioning to a new route. It was needed but we might want to revisit with nested routes.